### PR TITLE
created basic migrations and models

### PR DESCRIPTION
**Pull request by D-sense**

### CODE REVIEW

File: server/database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -15,10 +15,15 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('name')->unique();
```
Comment (thecodearcher) [inline server/database/migrations/2014_10_12_000000_create_users_table.php]: name can not be unique

File: server/database/migrations/2018_11_06_090909_create_posts_table.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('user_id');        // the member that is being followed by the current user
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->string('category')->default("general");
```
Comment (thecodearcher) [inline server/database/migrations/2018_11_06_090909_create_posts_table.php]: What if this is an enum? What do you think

File: server/database/migrations/2018_11_06_090909_create_posts_table.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            $table->text('content');
```
Comment (thecodearcher) [inline server/database/migrations/2018_11_06_090909_create_posts_table.php]: this should be renamed to body

Comment (thecodearcher) [reply server/database/migrations/2018_11_06_090909_create_posts_table.php]: remember to also change it in the $fillable

File: server/database/migrations/2018_11_06_090909_create_posts_table.php (hunk 2 above)
Comment (D-sense) [reply server/database/migrations/2018_11_06_090909_create_posts_table.php]: Sorry. What do you mean?

Comment (thecodearcher) [reply server/database/migrations/2018_11_06_090909_create_posts_table.php]: for the "category" can we use enum instead so we declare the categories as array of strings.

**Pull request merged**